packages/jest-auto-spies/src/create-function-spy.ts

```diff
@@ -19,6 +19,7 @@ function jestFunctionSpyFactory(name: string, spyFunctionImpl: Func) {
   return {
     functionSpy,
     objectToAddSpyMethodsTo: functionSpy,
+    functionSpyName: name,
```

Comment:
Do you need to return the functionSpyName here?
Or it it just left overs from experiments?

Reply:
yes, this seems a left over, I'll remove it